scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -227,7 +228,7 @@ public static BuildTargetResult PublishCoreHostPackages(BuildTargetContext c)
         {
             foreach (var file in Directory.GetFiles(Dirs.CorehostLocalPackages, "*.nupkg"))
             {
-                var hostBlob = $"{Channel}/Binaries/{CliNuGetVersion}/{Path.GetFileName(file)}";
+                var hostBlob = $"{Channel}/Binaries/{SharedFrameworkNugetVersion}/{Path.GetFileName(file)}";
```

Comment:
Hmm this is not good, perhaps LatestHostVersion would be more appropriate.

What I'm worried about here is that the host versions and sharedframework versions are different.